package.json

```diff
@@ -4,12 +4,18 @@
   "description": "WordPress Theme framework",
   "main": "index.js",
   "scripts": {
+	"bs": "browser-sync start --config build/browser-sync.config.js",
     "css": "npm-run-all css-compile css-postcss css-minify",
     "css-compile": "sass --style expanded --source-map --embed-sources --no-error-css sass/theme.scss:css/theme.css sass/custom-editor-style.scss:css/custom-editor-style.css",
     "css-minify": "cleancss -O1 --format breakWith=lf --with-rebase --source-map --source-map-inline-sources --output css/ --batch --batch-suffix \".min\" \"css/*.css\" \"!css/*.min.css\" \"!css/*rtl*.css\"",
-    "css-postcss": "postcss --config postcss.config.js --replace \"css/*.css\" \"!css/*.rtl*.css\" \"!css/*.min.css\"",
-    "watch": "npm-run-all --parallel watch-*",
-    "watch-css": "nodemon --watch sass/ --ext scss --exec \"npm-run-all css\""
+    "css-postcss": "postcss --config build/postcss.config.js --replace \"css/*.css\" \"!css/*.rtl*.css\" \"!css/*.min.css\"",
```

Comment:
we need a `postcss.config.js`

Reply:
It's in the `build/` folder. This was another convention from BS that I thought made sense, putting as much of the config files in a separate folder. When we submit to wp.org repo, we have to remove everything build related, so having them in one folder seems helpful.

Reply:
But the term build in my eyes have to do with the final build not general build. but that is just my POV.
I have no issue with having config files in the root. But if its a convention thing sure :)

Reply:
Yeah I keep going back and forth on this. I don't really like having a new directory but I also would love to have all the dev-type stuff collected together so we can more easily prepare for wp.org...

Reply:
do we actually need to clean the dev stuff out for wp.org?
If so then put that folder in /src as well

Reply:
That's a good point- maybe /src makes more sense and gives us one less directory. Or we just put it back in the root since that's pretty standard and we have to delete a lot of stuff from root anyway. 

We don't have to remove asset files (like scss) but we have to remove any config files as shown here https://make.wordpress.org/themes/handbook/review/required/#9-files